src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
```

Comment:
It's better to use a variable for the gap (e.g., $gap: 1px) to make it consistent and easy to update in the future.